### PR TITLE
Refactor Spelunky Classic HD

### PR DESCRIPTION
**Pull request by JeodC**
Use gmloader not box86

### CODE REVIEW

**Review by Cebion — APPROVED** [review]

File: ports/spelunky/spelunky/port.json
Comment (kloptops) [inline ports/spelunky/spelunky/port.json:1]: Why is this file added when it should already be in the repo? 🐢

**Review by kloptops — APPROVED** [review]
Looks good, but some files are weirdly being added when they already exist on the repo?
